engine/table/src/main/java/io/deephaven/engine/table/impl/WouldMatchOperation.java

```diff
@@ -517,13 +532,14 @@ private RowSet update(RowSet added, RowSet removed, RowSet modified,
 
             try (final SafeCloseableList toClose = new SafeCloseableList()) {
                 // Filter and add addeds
-                final WritableRowSet filteredAdded = toClose.add(filter.filter(added, source, table, false));
+                final WritableRowSet filteredAdded = toClose.add(filter.filter(added, table.getRowSet(), table, false));
```

Comment:
Was this incorrect before? Curious if `table.getRowSet` is semantically more appropriate?

Reply:
Yes, it was incorrect. If you had a filter that used "i" in a wouldMatch, the index operations would fail with an assertion, because you need to pass in the fullset and the source is not actually correct because added is not included in it at this point.

Reply:
Also worth noting, that I've disabled the virtual variables and arrays for would match, because they are problematic in terms of the values being calculated correctly without significant additional complexity.